app/models/service.rb

```diff
@@ -0,0 +1,8 @@
+class Service < ActiveRecord::Base
+  belongs_to :student
+  belongs_to :educator
+  belongs_to :service_type
+
+  validates_presence_of :educator_id, :student_id, :event_note_type_id, :recorded_at
```

Comment:
`event_note_type_id` looks like a leftover copy-pasta, this should be `:service_type_id`.